docs/configuration.md

```diff
@@ -151,6 +151,13 @@ of the most common options to set are:
     Number of cores to use for the driver process, only in cluster mode.
   </td>
 </tr>
+<tr>
+  <td><code>spark.driver.resource.gpu.count</code></td>
+  <td>0</td>
+  <td>
+    Number of gpus to use for the driver process, only in cluster mode.
+  </td>
+</tr>
```

Comment:
we need to add in the spark.driver.resource.gpu.discoveryScript and spark.driver.resource.gpu.addresses here